frontend/src/main/scala/bloop/engine/tasks/RunTasks.scala

```diff
@@ -0,0 +1,38 @@
+package bloop.engine.tasks
+
+import bloop.Project
+import bloop.cli.ExitStatus
+import bloop.engine.State
+import bloop.exec.ProcessConfig
+
+import sbt.internal.inc.Analysis
+import xsbt.api.Discovery
+
+object RunTasks {
+  def run(state: State, project: Project, className: String, args: Array[String]): State = {
+    val classpath = project.classesDir +: project.classpath
```

Comment:
We're doing this a lot in lots of places. Maybe it would be a good idea to create a method that creates the classpath out of the project and update all the call sites.

Reply:
Sounds good 👍